optaplanner-test/src/test/java/org/optaplanner/test/api/score/stream/testdata/TestdataConstraintVerifierEntity.java

```diff
@@ -0,0 +1,35 @@
+package org.optaplanner.test.api.score.stream.testdata;
+
+import org.optaplanner.core.api.domain.entity.PlanningEntity;
+import org.optaplanner.core.api.domain.variable.PlanningVariable;
+import org.optaplanner.core.impl.testdata.domain.TestdataObject;
+import org.optaplanner.core.impl.testdata.domain.TestdataValue;
+
+@PlanningEntity
+public final class TestdataConstraintVerifierEntity extends TestdataObject {
```

Comment:
If we call it `TestdataConstraintVerifierFirstEntity`, any reader will immediately realize there is also a second one.